src/content/getting-started/quickstart/openshift/rhos/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "Hybrid OpenStack and AWS"
+weight: 50
+---
+
+This quickstart guide covers the necessary steps to deploy two OpenShift Container Platform (OCP) clusters: one on AWS
+and the other one on OpeStack, both with full stack automation, also known as installer-provisioned infrastructure (IPI).
+Once the OpenShift clusters are deployed, we deploy Submariner with Service Discovery to interconnect the two clusters.
+Note that this guide focuses on Submariner deployment on clusters with non-overlapping Pod and Service CIDRs.
+For connecting clusters with overlapping CIDRs, please refer to the
+[Submariner with Globalnet guide](../globalnet/).
+
+{{< include "/resources/shared/openshift/setup_openshift_openstack.md" >}}
+
+{{% notice info %}}
+Please ensure that the tools you downloaded above are compatible with your OpenShift Container Platform version. For more information,
+please refer to the official [OpenShift documentation](https://docs.openshift.com/container-platform/).
+{{% /notice %}}
+
+{{< include "/resources/shared/openshift/create_clusters_aws_openstack.md" >}}
+
+### Install `subctl`
+
+{{% subctl-install %}}
+
+### Prepare OpenStack and AWS Clusters for Submariner
+
+{{% cloud-prepare/intro %}}
+{{% cloud-prepare/openstack %}}
```

Comment:
So this "openstack" is being flagged because it's not in a `code block`. Maybe we can just make the file follow the naming convention, be `OpenStack`?

Reply:
Done.